examples/complicated/validators.go

```diff
@@ -30,7 +30,10 @@ func (r AliasArray) Validate() error {
 		_ = i
 		_ = x
 	}
-	return &errs
+	if errs.HasErrors() {
```

Comment:
I did it in the first approach.
 did it wrong:
`func (e *ErrList) ErrorOrNil() *ErrList`
I want to evidently return `ErrList` from `Validate` method, so in consumer I can cast to ErrList.
Will try your suggestion.